brd-ios/breadwallet/src/Redesign/Scenes/Swap/SwapMain/SwapMainViewController.swift

```diff
@@ -0,0 +1,146 @@
+//
+//  SwapMainViewController.swift
+//  breadwallet
+//
+//  Created by Kenan Mamedoff on 05/07/2022.
+//  Copyright © 2022 Fabriik Exchange, LLC. All rights reserved.
+//
+//  See the LICENSE file at the project root for license information.
+//
+
+import AVFoundation
+import UIKit
+
+class SwapMainViewController: BaseTableViewController<KYCCoordinator,
+                              SwapMainInteractor,
+                              SwapMainPresenter,
+                              SwapMainStore>,
+                              SwapMainResponseDisplays {
+    
+    typealias Models = SwapMainModels
+    
+    override var sceneLeftAlignedTitle: String? {
+         // TODO: localize
+        return "Swap"
+    }
+    
+    // TODO: Get rid of those if possible.
+    private var fromFiatAmount: SwapMainModels.Amounts.CurrencyData?
+    private var fromCryptoAmount: SwapMainModels.Amounts.CurrencyData?
+    private var toFiatAmount: SwapMainModels.Amounts.CurrencyData?
+    private var toCryptoAmount: SwapMainModels.Amounts.CurrencyData?
```

Comment:
def get rid of them... they already exist on ur dataStore... which VC has access 2.. if needed